### ISSUE TITLE
Accessibility errors: Missing label

### ISSUE THREAD

**Issue by SirFischer** [AdelRedaa97/react-native-select-dropdown#129]
I haven't found a way to add and connect a label to the drop down. How should i go about it with the today's version of the dropdown, or is it missing this feature?

**Comment by Centeos** [comment]
Facing the same issue.

**Comment by SirFischer** [comment]
The problem seems to be that the modal doesn't have an accessibilityLabel which the google play pre-launch report is complaining about. Could you expose that field?

**Comment by DanielCeron-dc** [comment]
#156

**Comment by AdelRedaa97** [comment]
Added in v4.0 take a look at the changes made in v4.0
https://github.com/AdelRedaa97/react-native-select-dropdown?tab=readme-ov-file#-major-changes

Feel free to open the issue again if it still exists
